lib/galaxy/schema/__init__.py

```diff
@@ -1,13 +1,44 @@
-import typing
+from typing import (
+    Dict,
+    List,
+    Optional,
+)
 
 from pydantic import BaseModel
 
 
 class BootstrapAdminUser(BaseModel):
     id = 0
-    email: typing.Optional[str] = None
-    preferences: typing.Dict[str, str] = {}
+    email: Optional[str] = None
+    preferences: Dict[str, str] = {}
     bootstrap_admin_user = True
 
     def all_roles(*args) -> list:
         return []
+
+
+class Tour(BaseModel):
+    id: str
+    name: str
+    description: str
+    tags: List[str]
+
+
+class TourList(BaseModel):
+    __root__: List[Tour] = []
+
+
+class TourStep(BaseModel):
+    title: Optional[str] = None
```

Comment:
Can you add a description to these fields ?
Something like
```suggestion
    title: Optional[str] = Field(title="Title", description="The title of this tour step", example="Click here")
```
?

Reply:
Done.